edgexfoundry/device-virtual-go
# build(deps): bump device-sdk-go

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/device-virtual-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  <link to docs PR>

## Testing Instructions

1. Run `device-virtual` from this branch
2. Verify device service is working correctly

## New Dependency Instructions (If applicable)
